flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/catalog/JavaCatalogTableTest.java

```diff
@@ -177,6 +177,31 @@ void testTimeAttributeOfView() {
                         + "GROUP BY window_start, window_end");
     }
 
+    @TestTemplate
+    void testTimeAttributeOfViewSelect() {
+        if (!streamingMode) {
+            // time attributes not supported in batch
+            return;
+        }
+        TableTestUtil testUtil = getTestUtil();
+        TableEnvironment tableEnvironment = testUtil.getTableEnv();
```

Comment:
someone in our team tested and the test works before the fix. Is there a test that would have failed before the fix and now passes after your fix?

Reply:
Please ask them to verify again, cause it does not pass without the changes here, if you have https://issues.apache.org/jira/browse/FLINK-36883 applied.